sdk/aqueduct/backend/api_client.py

```diff
@@ -80,7 +80,7 @@ class APIClient:
     GET_DYNAMIC_ENGINE_STATUS_ROUTE = "/api/resource/dynamic-engine/status"
     EDIT_DYNAMIC_ENGINE_ROUTE_TEMPLATE = "/api/resource/dynamic-engine/%s/edit"
 
-    GET_IMAGE_URL_ROUTE = "/api/resourece/container-registry/url"
```

Comment:
do you see any other this typo in the code base?

Reply:
nope